src/components/Auth/Login/Login.type.ts

```diff
@@ -0,0 +1,4 @@
+export interface ILogin {
+  email: string;
+  password: string;
```

Comment:
We should keep consistency with file naming

Reply:
I confirm